src/main/java/com/thindeck/dynamo/DyContext.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2014-2015, Thindeck.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the thindeck.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.thindeck.dynamo;
+
+import com.jcabi.aspects.Immutable;
+import com.thindeck.api.Context;
+import com.thindeck.api.Memo;
+import java.io.IOException;
+import java.util.logging.Level;
+
+/**
+ * Dynamo implementation of the {@link com.thindeck.api.Context}.
+ * @author Mauricio Herrera (oruam85@gmail.com)
+ * @version $Id$
+ */
+@Immutable
+public final class DyContext implements Context {
+    // @todo #462:30min  DyContext.memo() should be implemented. This method
+    // should  return the Memo object that will use in this context.
+    @Override
+    public Memo memo() throws IOException {
+        throw new UnsupportedOperationException("#memo");
+    }
+
+    // @todo #462:30min DyContext.log() should be implemented. This method
+    // should add a new line in the log.
```

Comment:
@oruam85 the `s` for `should`. puzzle lines after the first one need a whitespace.